ssl/statem/extensions.c

```diff
@@ -83,6 +113,13 @@ typedef struct extensions_definition_st {
      * protocol versions
      */
     unsigned int context;
+#ifndef OPENSSL_NO_ECH
+    /*
+     * TODO(ECH): it might be better to not conditionally compile this
+     * field so we don't need ifndefs all over the ext_defs values
```

Comment:
+1 to that

Reply:
yeah, did that - if needed can put 'em all back in later